config/samples/testing_v1alpha1_testdefinition.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: testing.kyma-project.io/v1alpha1
+kind: TestDefinition
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: test-kubeless
+spec:
+  skip: n
+  disable_concurrency: n
+  components:
+    - kubeless
+    - service-catalog
+    - service-catalog-addons
+  timeout: 3m
+  pod_spec:
```

Comment:
Maybe we could follow Deployment definition and put here `template` for whole pod resource, not only its spec.

Reply:
I thought about it, too. Under the hood the template is adding only the metadata to pod but it can be helpful if ur test pod will need some required additional annotations